conans/test/util/pkg_config_test.py

```diff
@@ -104,3 +105,21 @@ def rpaths_libs_test(self):
                         "-Wl,-rpath=/my_prefix/path/lib2")
             pkg_config = PkgConfig("MyLib")
             self.assertIn(expected, " ".join(lib for lib in pkg_config.libs))
+
+    def add_libraries_from_pc_test(self):
+        if platform.system() == "Windows":
+            return
```

Comment:
Use the decorator `@unittest.skipUnless(platform.system() == "Windows", "Requires pkg-config")` instead of returning from the test.